### ISSUE TITLE
After updating game files to 3652, watching a replay reinstalls 3650

### ISSUE THREAD

**Issue by GuyGuyGuy** [FAForever/fa#1235]
As per title, watching a replay causes 3650 game files to be reinstalled rather than using the new 3652 files that have already been installed.

**Comment by aeoncleanse** [comment]
Yes this is normal. Watching a replay from a lower, older replay will re-load that version's files.

**Comment by GuyGuyGuy** [comment]
This happens even when watching the replay of a 3652 game.
Apologies, should have made that clear.

**Comment by aeoncleanse** [comment]
Ah...

**Comment by aeoncleanse** [comment]
@Sheeo ?

**Comment by GuyGuyGuy** [comment]
Dude it's only been 7 mins since I posted, give him a break xD

**Comment by Sheeo** [comment]
Right, sec.

**Comment by Sheeo** [comment]
Please confirm that this works now

**Comment by GuyGuyGuy** [comment]
For _**new**_ replays this now works correctly.
Whilst you were making the change, I was in the middle of the game. After the game I restarted FAF. The replay of the game I was playing still loads 3650 files, but if I now make a new game (I ran a quick test) and then watch the replay of that, it runs with 3652 files.
So replays of games played between the time that 3652 was released and the time you fixed it today may be broken.
